src/main/kotlin/dk/cachet/carp/webservices/study/service/CoreRecruitmentService.kt

```diff
@@ -19,10 +20,11 @@ import org.springframework.stereotype.Component
 class CoreRecruitmentService
 (
     participantRepository: CoreParticipantRepository,
-    coreDeploymentService: CoreDeploymentService,
     coreEventBus: CoreEventBus,
+    private val coreDeploymentService: CoreDeploymentService,
+    private val idataStreamService: IDataStreamService,
```

Comment:
likewise, you don't need the i prefix

Reply:
sticking to original naming IDataStreamService, IDataStreamServiceImpl.

Reply:
yeah but the i is not necessary in the variable name :D

Reply:
But sure, we can fix all this naming convention hell once we go through with the most important stuff